polling_stations/apps/addressbase/models.py

```diff
@@ -63,6 +65,29 @@ def polling_station(self):
             return station[0]
         return None
 
+    @property
+    def polling_station_with_elections(self):
+        """
+        Returns the polling station for this address,
+        with the elections field from the most recent data import event
+        for the relevant council
+        """
+        try:
+            elections = Subquery(
+                DataEvent.objects.filter(
+                    council_id=OuterRef("council_id"),
+                )
+                .order_by("-created")
+                .values("election_dates")[:1]
+            )
```

Comment:
Should this subquery be filtering on `event_type`

Reply:
I think so, to guard against any future event types. It's not going to add any overhead to the query, is it?